lesson40.1/react-progect/src/components/Products.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState, useEffect, createContext } from 'react';
+import '../style/Products.css';
+import Product from './Product';
+import CreateProduct from './CreateProduct';
+
+export const ProductContext = createContext();
+
+function Products({ addToCart }) {
```

Comment:
you have `ProductContext` now, you don't need to pass `addToCart` to props just pass it to children
this is the reason why we use Context

and you can have your context on one level above to be able set `addToCart` to global context